src/Symfony/Component/Routing/Loader/schema/routing/routing-1.0.xsd

```diff
@@ -24,6 +24,14 @@
     </xsd:choice>
   </xsd:complexType>
 
+  <xsd:complexType name="localised-path">
```

Comment:
`localized-path` as we use AE anywhere else?

Reply:
fixed in 2004092d